apps/web/package.json

```diff
@@ -12,7 +12,7 @@
     "jest": "jest src",
     "types": "sapper build && tsc && svelte-check --treshold error",
     "installCourse": "./scripts/installExternalCourse.sh",
-    "installAllExternalCourses": "python ./scripts/installExternalCourses.py ../../config/courses.json",
+    "installAllExternalCourses": "python3 ./scripts/installExternalCourses.py ../../config/courses.json",
```

Comment:
I think this is not correct, I think that Python 2 is considered legacy software at this point and it should be called `python2` while `python3` should be called `python`. 

However I think by replacing it with `poetry run python` I think it will be even managed through poetry, so then I think it should work even if it's an outdated OS that still considers Python2 to be the default version